site/_includes/partials/banner.njk

```diff
@@ -0,0 +1,7 @@
+
+<div class="wrapper banner gap-top-300 rounded-500 banner--{{ banner.type }}">
```

Comment:
probably should add an if there is a banner for the page before displaying it.

Reply:
done